docs/units/index.rst

```diff
@@ -212,7 +214,8 @@ Reference/API
 
 .. automodapi:: astropy.units.function
 
-.. automodapi:: astropy.units.photometric
+.. automodapi:: astropy.units.function.logarithmic
```

Comment:
One more thing... What is the reasoning that was behind the ordering of the API listed? It does not appear to be strictly alphabetical and I notice that `logarithmic` is rather "far way" from `photometric` despite their close relation in functionality.

Reply:
p.s. I actually tried to define the magnitudes associated with the items in photometry in that file, but then it turned out to be harder to get a good list, since if one uses `:include-all-objects:`, one then repeats all the variables defining the zero points.

Reply:
Okay. Let's leave it be then. Thanks for the clarifications!